### ISSUE TITLE
Is there a way to display the Context menu without the labels.

### ISSUE THREAD

**Issue by himanshu1721** [nandorojo/zeego#116]
![IMG_8463](https://github.com/user-attachments/assets/f503f5a6-047b-4d60-a204-59f2f1e08494)

Hello. I would like to say this is an amazing library and I was searching for something like this for a very very long time. 

I would like to know if there is a way to preview the context menu without the labels (just the background blur). One way is to use <.Preview />, but it centers the content. I want the content to remain in its original position, with only the background blur applied.

In short - 

Is there a way to achieve Image 2, without the 'Hello' item. 
I can achieve the Image 3, but it is placed in the center.

Again great library. Cheers.

Image 2
![IMG_8462](https://github.com/user-attachments/assets/df764cf0-f53f-4914-a1f5-bf754ebcbefe)

Image 3
![Screenshot 2024-12-05 at 4 37 37 PM](https://github.com/user-attachments/assets/443f2eac-aed6-4628-b1eb-3c84d78440a0)

**Comment by nandorojo** [comment]
Not that I know of, I haven’t seen this on iOS before. Users probably expect menu items.

**Comment by himanshu1721** [comment]
Just for the aesthetics or make it screenshot-worthy by highlighting a specific component on screen.

No issues.

**Comment by himanshu1721** [comment]
@nandorojo Hey Fernando, I was wondering if you can work on this feature whenever you are free. As I mentioned there might be cases where only blurring the background is required for UI.

My case is I need to post screenshots and do marketing around those screenshots. It would look amazing and will be used by devs for sure.

![Group 142 1](https://github.com/user-attachments/assets/8ecad846-b066-4a65-b369-19b6b423fe85)

**Comment by himanshu1721** [comment]
![Screenshot 2024-12-10 at 11 47 08 PM 1 (1)](https://github.com/user-attachments/assets/414c95fb-2d07-409e-8884-212013f0baaf)


@nandorojo A good example of my above point is iOS Notifications. Long pressing the notification does nothing just blurs the background.

**Comment by nandorojo** [comment]
This would have to get implemented in the upstream iOS library I believe

**Comment by himanshu1721** [comment]
Might worth the headache ;)

**Comment by nandorojo** [comment]
Feel free to open an issue on that repo

**Comment by himanshu1721** [comment]
I would have loved to. But I don't think I am skilled enough to contribute to a lib like this. In a few months maybe.

Cheers.

**Comment by nandorojo** [comment]
You could still open an issue on the other repo as a feature request
